### PR TITLE
✨ [FEAT] :  Ledger Sync Status

### PR DESCRIPTION
**Pull request by mcayuelas-ledger**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Add Ledger Sync Status check
- call to CloudSync
- Call to Trustchain

- ReactUI : Fix color (using palette) + add new icons

if both are successful then it's ok, else display an Error 

Screenshot 2024-07-26 at 12 11 44

https://github.com/user-attachments/assets/4c44b625-dc88-496c-911b-02359e456be4

Screenshot 2024-07-26 at 12 14 31

### ❓ Context

- **JIRA or GitHub link**:  [LIVE-13237]

https://www.figma.com/design/uI7PKnUK8nAZ0USyiEF3wQ/Wallet-Sync?node-id=9394-18034&t=bh2bNn13pNlILXon-0
---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-13237]: https://ledgerhq.atlassian.net/browse/LIVE-13237?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-desktop/src/newArch/features/WalletSync/components/AlertError.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Text, Alert } from "@ledgerhq/react-ui";
+
+type Props = { error: Error };
```
[the remark this answers is not in the document]
Comment (mcayuelas-ledger) [reply apps/ledger-live-desktop/src/newArch/features/WalletSync/components/AlertError.tsx:4]: wich function?

[the remark this answers is not in the document]
Comment (LucasWerey) [reply apps/ledger-live-desktop/src/newArch/features/WalletSync/components/AlertError.tsx:4]: the component*

[the remark this answers is not in the document]
Comment (LucasWerey) [reply apps/ledger-live-desktop/src/newArch/features/WalletSync/components/AlertError.tsx:4]: nvm it will make it messier

**Review by LucasWerey — APPROVED** [review]

**Review by LucasWerey — DISMISSED** [review]

File: apps/ledger-live-desktop/src/newArch/features/WalletSync/hooks/useLedgerSyncInfo.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      queryFn: () => TrustchainApi.fetchStatus(),
+    },
+    {
+      queryKey: [QueryKey.fetchCloudSyncStatus],
+      queryFn: () => CloudSyncApi.fetchStatus(),
+    },
+  ];
+
+  return useQueries({
+    queries: QUERIES.map(query => query),
```
Comment (thesan) [inline apps/ledger-live-desktop/src/newArch/features/WalletSync/hooks/useLedgerSyncInfo.ts:21]: Did you forget this map or is it just to clone the array ?

Comment (mcayuelas-ledger) [reply apps/ledger-live-desktop/src/newArch/features/WalletSync/hooks/useLedgerSyncInfo.ts]: Nice catch :)

**Review by thesan — APPROVED** [review]

**Pull request merged**